pages/docs/widgets/variants/button.en.mdx

```diff
@@ -0,0 +1,222 @@
+
+You can find your token on the settings page in your Beeport account.   
+If you don't have a Beeport account, you can request a token with this form:
+
+<RequestToken />
+
+---
+
+#### Optional parameters
+
+##### `brand` string
+
+The brand/manufacturer of the equipment.
+
+```html
+<a data-brand="Caterpillar">
+```
+
+##### `model` string
+
+The model of the equipment.
+
+```html
+<a data-model="AS 450">
+```
+
+##### `year` integer
+
+The year the equipment was built.
+
+```html
+<a data-year="2017">
+```
+
+##### `purchase-price` decimal
+
+The amount it costs to buy the equipment. This value is shown in an input field, and
+can be changed by the user.   
+It is a required parameter so the widget can check whether the object is eligible for leasing.
+
+⚠ The equipment should cost at least € 15.000. We don’t accept deals with a lower purchase price.
+
+```html
+<a data-purchase-price="15000">
+```
+
+##### `category-id` integer 
+
+With the category ID you can supply the asset category the equipment belongs to. Lookup the appropriate ID in 
+the [category table](#available-categories).   
+If you do not supply the category ID, a dropdown will be rendered where the user must choose a category.
+
+```html
+<a data-category-id="3">
+```
+
+##### `object-used` boolean
+
+Indicates the condition the equipment is in. `true` if it is used, `false` if it is new.   
+Defaults to `true`.
+
+```html
+<a data-object-used="false">
+```
+
+##### `width` integer
+
+The width of the rendered widget, in pixels or a percentage. Defaults to 180 pixels.
+
+```html
+<a data-width="400">
+```
+
+##### `height` integer
+
+The height of the rendered widget, in pixels or a percentage. Defaults to 40 pixels.
+
+```html
+<a data-width="600">
+```
+
+##### `button-color` string
+
+This will set the text color of the button to the given value. The value
+can be any valid CSS color. Defaults to `#fff`.
+
+```html
+<a data-button-color="#056fff">
+```
+
+##### `button-background-color` string
+
+This will set the background color of the button to the given value. The value
+can be any valid CSS color. Defaults to `#ffa100`.
+
+```html
+<a data-button-background-color="rgb(0, 0, 0)">
+```
+
+##### `text` string
+
+This will change the text of the button. Defaults to “Bereken je lease”.
+
+```html
+<a data-button-background-color="rgb(0, 0, 0)">
+```
+
+---
+
+### Available categories
+
+These are the available options for the `category-id` parameter.
+
+ID | Description in Dutch | Description in English
+-- | -------------------- | ----------------------
+1 | Energie | Energy
+2 | Grondverzet | Earthmoving
+3 | Wegtransport | Road transport
+4 | Containers | Containers
+5 | Bouw | Construction
+6 | Heftrucks | Forklifts
+7 | Kranen | Cranes
+8 | Agrarisch | Agriculture
+11 | Haven | Harbour
+12 | Recycling | Recycling
+13 | Maritiem | Maritime
+15 | Metaalbewerking | Metal processing
```

Comment:
Spoor, Werkboten, Binnenvaart, Schoonmaakmachines en SPMT (wat dit ook moge zijn) zijn bewust weggelaten? Deze zijn volgens mij later toegevoegd en denk ook niet relevant voor de widget?

Reply:
Dat is niet bewust, ik zal ze nog even toevoegen.

SPMT is een hoop wielen 😛 

![image](https://user-images.githubusercontent.com/596975/142638668-bdc20bf9-7636-4035-b5aa-7a572b3ec3ce.png)

https://nl.wikipedia.org/wiki/SPMT_(voertuig)